rc/.eslintrc

```diff
@@ -1,3 +1,10 @@
 {
-  "extends": ["standard", "standard-react"]
+  "extends": ["standard", "standard-react"],
+  "rules": {
+    "standard/object-curly-even-spacing": [2, "either"],
```

Comment:
Agreed,  as a user of `eslint-config-standard` also,  I still don't know the answer.  @Flet thoughts?